cmd/pomerium/main.go

```diff
@@ -247,8 +251,20 @@ func setupTracing(opt *config.Options) error {
 
 func setupHTTPRedirectServer(opt *config.Options, wg *sync.WaitGroup) error {
 	if opt.HTTPRedirectAddr != "" {
-		serverOpts := httputil.ServerOptions{Addr: opt.HTTPRedirectAddr}
-		srv, err := httputil.NewServer(&serverOpts, httputil.RedirectHandler(), wg)
+		serverOpts := httputil.ServerOptions{
+			Addr:              opt.HTTPRedirectAddr,
+			Insecure:          true,
+			Service:           "HTTP->HTTPS Redirect",
+			ReadHeaderTimeout: 5 * time.Second,
```

Comment:
Changed default timeouts to be stricter given this endpoint will be used exclusively for redirects and http acme challenges.